### PR TITLE
Add isLeapYear question.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: assets/questions/isLeapYear.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    @classmethod
+    def handleDivisibleByFourCaseOnly(cls, year):
+        return year % 4 == 0
+`
+  },
+  tasks: [{
+    id: 'isLeapYear',
+    instructions: [{
+      content: [
+        'Three criteria must be taken into account to identify leap years:'
```
Comment (eyurko) [inline assets/questions/isLeapYear.js]: The wording seems very ... formal, here. What about:

"To determine whether or not a given year is a leap year, we must consider three criteria:"

File: assets/questions/isLeapYear.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        allowedOutputs: [false]
+      }, {
+        input: 1905,
+        allowedOutputs: [false]
+      }, {
+        input: 702,
+        allowedOutputs: [false]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_FOUR_BUT_NOT_HUNDRED',
```
Comment (eyurko) [inline assets/questions/isLeapYear.js]: BUT_NOT_ONE_HUNDRED

File: assets/questions/isLeapYear.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        allowedOutputs: [true]
+      }, {
+        input: 2004,
+        allowedOutputs: [true]
+      }, {
+        input: 2016,
+        allowedOutputs: [true]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_HUNDRED_BUT_NOT_FOUR_HUNDRED',
```
Comment (eyurko) [inline assets/questions/isLeapYear.js]: ONE_HUNDRED

ditto elsewhere; I think it's more clear.

File: assets/questions/isLeapYear.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+          'forget to include a "return" statement?'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.returnTrue',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          "It looks like you're consistently returning True for all test ",
+          "cases. This implies that every year is a leap year, but that's ",
```
Comment (eyurko) [inline assets/questions/isLeapYear.js]: This implies that every year is a leap year, which is not true.

File: assets/questions/isLeapYear.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+          "not true..."
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.returnFalse',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          "It looks like you're consistently returning False for all test ",
+          "cases. This implies that there are no leap years, but that's not ",
```
Comment (eyurko) [inline assets/questions/isLeapYear.js]: This implies there are no leap years, but that is not true.

File: assets/questions/isLeapYear.js (hunk 1 above)
Comment (seanlip) [reply assets/questions/isLeapYear.js]: Ah, I just followed the original question. I do prefer the updated wording, and have changed it -- thanks!

File: assets/questions/isLeapYear.js (hunk 2 above)
Comment (seanlip) [reply assets/questions/isLeapYear.js]: Done.

File: assets/questions/isLeapYear.js (hunk 3 above)
Comment (seanlip) [reply assets/questions/isLeapYear.js]: Done.

File: assets/questions/isLeapYear.js (hunk 4 above)
Comment (seanlip) [reply assets/questions/isLeapYear.js]: Done.

(Also, I thought "but" was the issue here, but we seem to have preserved it below, so just wanted to check what the issue was. Was it the ellipses? They were originally there to signal to the student "now it's your turn to think and continue the chain of reasoning", but I'm fine with removing them.)

File: assets/questions/isLeapYear.js (hunk 5 above)
Comment (seanlip) [reply assets/questions/isLeapYear.js]: Done. (Just for my edification, what was the primary issue with the original here?)

Comment (eyurko) [reply assets/questions/isLeapYear.js]: It was the ellipses, yeah. I prefer a more authoritative tone when we're trying to like, tell them what's wrong. Trailing off seems like we're unsure. Or that's the impression I get.

**Pull request merged**